src/utils.ts

```diff
@@ -237,26 +240,20 @@ export function sanatizeInputFiles(inputs: any, log: Logger): string[] {
  * @param chain
  * @param repository
  */
-export function findByAddress(address: string, chain: string, repository: string): Match[] {
-  const path = `${repository}/contract/${chain}/${address}`
-  const normalizedPath = require("path").join(__dirname, '..', path);
-  const files = [];
-
-  const matches: Match[] = [];
-
-  fs.readdirSync(normalizedPath).forEach((file) => {
-    files.push(file)
-  });
-
-  if(files.length > 0){
-    matches.push({
-      address: address,
-      status: "perfect"
-    });
-    return matches
+export function findByAddress(address: string, chain: string, repository: string): Match {
+  const addressPath = `${repository}/contract/${chain}/${address}`
+  const normalizedPath = path.join(__dirname, '..', addressPath);
+
+  try {
+    fs.readdirSync(normalizedPath);
+  } catch(e){
+    throw new Error("Address not found in repository");
   }
 
-  throw new Error("Address not found in repository");
+  return {
+    address: address,
+    status: "perfect"
+  }
```

Comment:
I've changed this to only return a single object, because there aren't multiple addresses to be found.

Reply:
Reverted in 72ecb77